src/pages/App/App.tsx

```diff
@@ -4,6 +4,9 @@ import { Home, ThemePresentation, Login, Dashboard } from "@pages";
 import { Theme } from "@utils";
 import { Routes } from "react-router";
 import { BrowserRouter as Router, Route } from "react-router-dom";
+import {logBuffer} from "../../shared/logBuffer.config";
```

Comment:
@DimitriB01 hier am besten wieder "@shared"

Reply:
mag er nicht :/
Cannot find module '@shared/logBuffer.config' or its corresponding type declarations.

Reply:
Hast die config richtig in der index.ts exportiert?

Reply:
🤦‍♂️